### DISCUSSION TITLE
Infos über den Cache im neuen Logform

### DISCUSSION THREAD

**Discussion by ChristianGK-GC** [2Abendsegler/GClh#2389] [Ideas]
Ich habe seit einiger Zeit bei mir eine kleine Erweiterung laufen. 
Vielleicht passt die auch in den GClh.
Ich habe immer vermisst, dass man sieht um was für einen Cache es sich handelt und welche Wertung der Cache hat.
Basierend auf den Infos packe ich unterschiedliche TBs zum Log dazu oder nicht.

![grafik](https://user-images.githubusercontent.com/24353480/231716364-5f71a9ee-7c0b-4293-abec-35d6a05eb5ce.png)

https://github.com/2Abendsegler/GClh/compare/collector...ChristianGK-GC:GClh:collector

**Comment by 2Abendsegler** [comment]
Moin,

ja, gute Idee. 

Es erscheint mir noch netter, wenn wir die D und T Werte mit einem Icon versehen, so wie es im Dashboard gemacht wird. 

Und den Cache Typ, auch als Icon, könnten wir vor den Cache Namen setzen, so wie es anderswo auch gemacht wird.

Also etwa so:
![unbenannt](https://user-images.githubusercontent.com/22332216/232194183-a2dd0a3f-45c0-4350-816f-f402fa47229a.jpg)

Dann könnten wir die Icons auch mit einem passenden title versehen, so wie wir es bei den bisherigen Informationen machen. Und alle Informationen wären direkt und auch auf Android Geräten zugänglich.

Womöglich müssen wir noch beachten, dass der Cache Name zu lang wird und umbricht. Das hatte ich bei den bisherigen Informationen vergessen, scheint jetzt mit den zusätzlichen Informationen in der Zeile aber noch wichtiger zu werden. Wenn der Name also umbricht, dann sollten die Informationen so bestehen bleiben. Jetzt scheinen sie bei langen Namen nach unten gedrückt zu werden, habe ich aber nur mal kurz durch eine Verlängerung eines Namens angetestet. Ich weiß gar nicht ob ein Name wirklich so lang werden kann, dass er umbricht.

**Comment by ChristianGK-GC** [comment]
Da gefällt mir deine Integration deulich besser.
Ich habe es mit den Sternchen und dem Icon versucht, da war es dann zu lange.
Drum bin ich gleich auf die komakte Lösung gegangen.

**Comment by 2Abendsegler** [comment]
Ich habe das noch nicht integriert, ich habe nur Bildchen zusammen kopiert. 
Ich habe gehofft, dass du das entwickeln möchtest, weil ich im Moment keine Zeit habe. Was meinst du?

**Reply by ChristianGK-GC** [reply]
Ich kann das gern mal probieren.
Wo hast du denn die Bildchen her.
Die kenne ich gar nicht ;-)

**Reply by 2Abendsegler** [reply]
Im neuen Dashboard im Bereich "Recently Viewed":
![Unbenannt](https://user-images.githubusercontent.com/22332216/232449255-b5c49d5c-9c4c-4152-8188-0a7144780298.JPG)
Und wir haben diese Icons auch selbst im Bereich "Unpublished Hides" eingebaut:
![Unbenannt2](https://user-images.githubusercontent.com/22332216/232450162-a9ad24ae-22a3-44c7-946b-39a555a47f00.JPG)

Ich bin jetzt aber etwas unsicher, ob wir diese Icons verwenden sollen, wenn du die nicht kennst. Und ich habe auch mal kurz geschaut, die scheinen auch nur im neuen Dashboard verwendet zu werden. Überall sonst, auch auf neueren Seiten, werden noch die Sterne oder ein D bzw. T angegeben. 
Ich überlasse es dir, ob du die Icons verwenden möchtest oder ob du anstatt dessen ein D bzw. T verwenden möchtest.

**Comment by ChristianGK-GC** [comment]
So richtig gut will es nicht klappen.
![grafik](https://user-images.githubusercontent.com/24353480/232609915-701cbcfc-6d3b-4fde-b4a0-5ab134b24d94.png)
Bereits ein Leerzeichen sorgt schon für den Umbruch. Da muss ich noch nicht mal die 0,5 Werte testen.
Mit den Sternchen leider das gleiche Problem, der Platz.
![grafik](https://user-images.githubusercontent.com/24353480/232610149-433d8cd2-d640-4b09-968f-de54fd30d729.png)

**Reply by ChristianGK-GC** [reply]
Das finde ich die Beste Version.
![grafik](https://user-images.githubusercontent.com/24353480/232614943-986b795f-23f8-47f9-872c-03f7593b33e4.png)
oder
![grafik](https://user-images.githubusercontent.com/24353480/232615647-e20e4ada-d24e-4830-8b3e-e4538dc071ca.png)

**Comment by 2Abendsegler** [comment]
Deshalb hatte ich ja geschrieben, dass wir dafür sorgen müssen, dass die zusätzlichen Informationen auch beim Umbrechen bestehen bleiben. Das könnte dann so aussehen:
![Unbenannt1 1](https://user-images.githubusercontent.com/22332216/232646466-5804374d-668c-41d7-86a4-3d87bcaea91a.jpg)

Oder so, wie wir es bei zu langen Cacher Namen in den Logs im Listing machen. Im title könnte dann der komplette Name stehen.
![Unbenannt1 2](https://user-images.githubusercontent.com/22332216/232646584-6e0ebb80-88e2-45a7-8de9-1204455c83e3.jpg)

Mir ist noch aufgefallen, dass du scheinbar zuerst das T angibst und dann das D. Die richtige Reihenfolge wäre aber D T.

**Comment by ChristianGK-GC** [comment]
Du kannst es dir mal anschauen.
https://github.com/2Abendsegler/GClh/compare/collector...ChristianGK-GC:GClh:collector

Ich habe es mit 1000 Favoriten getestet.
D/T auch mit x.5
Der Platz sollte imho reichen.

![grafik](https://user-images.githubusercontent.com/24353480/232924830-75bf8748-5c32-43fb-a910-2286e9cb0fdb.png)
![grafik](https://user-images.githubusercontent.com/24353480/232925515-168a72a3-d57e-42f7-a073-3653598afb41.png)
![grafik](https://user-images.githubusercontent.com/24353480/232925586-3db589d2-1663-4e3e-9669-7476ef663d77.png)

Beim alten Log Form passt es auch.
![grafik](https://user-images.githubusercontent.com/24353480/232925638-4bc69a3b-24f3-45ef-ae78-0cda8483b884.png)

**Comment by 2Abendsegler** [comment]
Ja, das sieht sehr schön aus!

Ich generiere im Anschluss aus der Discussion ein Issue, dann kannst du bei deinem PR auf das Issue verweisen.

**Reply by 2Abendsegler** [reply]
Issue #2399 ist angelegt. Vielleicht kannst du dort kurz Bescheid geben, wenn der PR da ist, dann kann ich dich dort auch assignen.
